DemoqaProject/tests/WebTablesTest.cs

```diff
@@ -0,0 +1,50 @@
+using DemoqaProject.PageObjects;
+using NUnit.Framework;
+
+namespace DemoqaProject
+{
+    public class WebTablesTest : BaseSeleniumTest
+    {
+        [Test]
+        public void CRUDTest()
+        {
+            HomePage homePage = new HomePage();
+            homePage.NavigateToElementPage();
+            Elements elements = new Elements();
+            
+            //Filling form
+            elements.NavigateToWebTables(StoreElementTexts.firstName, StoreElementTexts.lastName, StoreElementTexts.email, StoreElementTexts.age, StoreElementTexts.salary, StoreElementTexts.department);
+            //Checking added user with name Denzel
+            Assert.IsTrue(elements.CheckName("Denzel"));
+            //Editing existing record
+            elements.EditRecord();
+            //Checking edite user with name Tom
+            Assert.IsTrue(elements.CheckName("Tom"));
```

Comment:
The same, CheckName should have a param that it should check, do not hardcode such.

Reply:
done